src/components/task-plan/homework/exercises.cjsx

```diff
@@ -182,7 +183,10 @@ ExerciseTable = React.createClass
 
   renderExerciseRow: (exerciseId, index, hasTeks) ->
     {section, lo, tagString} = ExerciseStore.getTagStrings(exerciseId)
-    content = ExerciseStore.getContent(exerciseId)
+    content = $("<span>#{ExerciseStore.getContent(exerciseId)}</span>")
```

Comment:
Can you use `document.createElement('span')` and `.innerHTML = ` instead of jQuery?